crawler.rb

```diff
@@ -1,14 +1,17 @@
 # frozen_string_literal: true
 
 require "fileutils"
+require "relaton_ieee"
 
 t1 = Time.now
 puts "Started at: #{t1}"
 
-system("git clone https://github.com/ietf-tools/relaton-data-ieee")
+token = ARGV.shift
+system("git clone https://oauth2:#{token}@github.com/ietf-ribose/ieee-rawbib.git ieee-rawbib")
```

Comment:
I'm not sure but probably `oauth2` has to be replaced with `x-access-token` i.e. 

```suggestion
system("git clone https://x-access-token:#{token}@github.com/ietf-ribose/ieee-rawbib.git ieee-rawbib")
```